packages/core/lib/commands/config.js

```diff
@@ -73,6 +77,9 @@ const command = {
       }
 
       return;
+    } else if (command.list) {
+      options.logger.log("; Truffle config values");
```

Comment:
Could you remove the `; ` from the beginning of this string? I changed it locally to look like this at the command line.

<img width="878" alt="Screen Shot 2021-12-02 at 10 36 53 AM" src="https://user-images.githubusercontent.com/14827965/144453355-41d7a0f4-86cf-436c-b7dc-3ded8ef76617.png">

Reply:
Removed. I just add `;` because I followed `npm config list` :D